### ISSUE TITLE
Issues with custom colors, zsh + keyboard shortcuts

### ISSUE THREAD

**Issue by maurcz** [pdbpp/pdbpp#478]
Hello! First of all I just want to thank you for all the hard work put into this package - huge fan of custom colors and I can't live without sticky mode anymore :)

Here's my custom `.pdbrc.py` configuration file:

```
import pdb

class Config(pdb.DefaultConfig):
    highlight = False
    sticky_by_default = True
    prompt = '[pdb++] '
    use_pygments = True
    pygments_formatter_class = "pygments.formatters.TerminalTrueColorFormatter"
    pygments_formatter_kwargs = {"style": "monokai"}
```

I'm used to install `pdbpp` from source (following directions I got from some comments in this repo) with `pip install git+git://github.com/pdbpp/pdbpp@master`. Here's my `pip freeze`: 

```
fancycompleter @ git+https://github.com/pdbpp/fancycompleter@8ae963eff76b327b7533f39f40d04eb2b99fa3fd
pdbpp @ git+git://github.com/pdbpp/pdbpp@b757794857f98d53e3ebbe70879663d7d843a6c2
Pygments==2.10.0
pyrepl @ git+https://github.com/pdbpp/pyrepl@f1e5513e069c8b2fac5380fa7869277a6360b65a
six==1.16.0
wmctrl==0.4
```

And as you can see I get colors normally: 

![image](https://user-images.githubusercontent.com/1469466/145240329-4a81379d-b322-4ccc-91e0-13a7992959cb.png)

However, recently I've started to get issues with certain keyboard shortcuts (I use zsh+powerlevel10k). i.e. `Home` or `End` won't work, as well as holding `Option` / `Control` and key arrows to navigate between words. Using tab completions within `pdbpp` will also yield weird results:

![image](https://user-images.githubusercontent.com/1469466/145240964-04d2a741-84bd-474b-9262-dc6b8c2adbd9.png)

I've narrowed this issue down to `pyrepl` and **uninstalling the package fixes the issues with the keyboard shortcuts**. I'm also aware that I might stumble across issues like this by installing packages from source, so I'm ok with this solution.

But if the issue is "fixed", then why am I here? Well, while doing some investigation I decided to give the version of `pdbpp` published to PyPi a try. Turns out that colors suddenly won't work anymore with the package published to PyPi, event though _I haven't touched my `.pdbrc` file_.  New `pip freeze`:

```
fancycompleter==0.9.1
pdbpp==0.10.3
Pygments==2.10.0
pyrepl==0.9.0
wmctrl==0.4
```

Colors are gone (but no keyboard shortcuts issues!):
![image](https://user-images.githubusercontent.com/1469466/145241601-c8b30b0a-0674-4bc7-be07-10a2f44d9637.png)

That's super weird, but digging into the problem I've noticed that a file called `pdb.py` was installed to my environment, and that has no code whatsoever related to Pygments:

![image](https://user-images.githubusercontent.com/1469466/145241833-ea260426-0bdb-4719-a35a-67cce34d3611.png)

In contrast, this is the tree I get if I install pdbpp **from source**:
![image](https://user-images.githubusercontent.com/1469466/145242016-172c5932-bb0e-4188-92ba-bf6dd233b1cf.png)

That file has the section that understand Pygments ( which I believe is essentially [this](https://github.com/pdbpp/pdbpp/blob/b757794857f98d53e3ebbe70879663d7d843a6c2/src/pdbpp.py#L857) code). 

Bottom line:

- `pdbpp` from source has issues with keyboard shortcuts on `zsh` due to `pyrepl`. Uninstalling `pyrepl` fixes the issue.
- `pdbpp` from PyPi won't work with my previous `.pdbrc.py` configs. I suspect it has to deal with the code being packaged in the whl.

Anyway, some weird stuff going on, sorry for the long post but just wanted to provide as much information as possible. I've debated over separating these into two issues, but ultimately settled for a single issue that explains the whole story of how I found the problems.

Let me know if you need any further info. Thanks again for the awesome work here!

**Comment by giladbarnea** [comment]
I had the problem with pyrepl ==0.8.5, then after `pip install -U pyrepl` to 0.9.0, the problem was gone. Maybe it's not removing pyrepl but something more nuanced

**Comment by jamilraichouni** [comment]
> I had the problem with pyrepl ==0.8.5, then after `pip install -U pyrepl` to 0.9.0, the problem was gone. Maybe it's not removing pyrepl but something more nuanced

Wow cool! Thank you. That solved this issue here for me: #52
